tests/e2e/defaultvmgroup_test.go

```diff
@@ -57,38 +65,106 @@ var _ = Suite(&DefaultVMGroupTestSuite{})
-	log.Printf("START: defaultvmgroup.TestAddVMToDefaultTenant")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.AddVMToVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.AddVMToVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to add vm to the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestAddVMToDefaultTenant")
+	misc.LogTestEnd(c.TestName())
 }
 
 // Test step:
 // Remove a vm from the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestRemoveDefaultTenantVMs(c *C) {
-	log.Printf("START: defaultvmgroup.TestRemoveDefaultTenantVMs")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.RemoveVMFromVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.RemoveVMFromVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to remove vm from the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestRemoveDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
 }
 
 // Test step:
 //  Replace a vm from the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestReplaceDefaultTenantVMs(c *C) {
-	log.Printf("START: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.ReplaceVMFromVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.ReplaceVMFromVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to replace vm from the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
+}
+
+// Verify volume creation after deleting _DEFAULT tenant
+// 1. Create a volume
+// 2. Verify volume from docker host and esx
+// 2. Delete default tenant
+// 3. Verify not able to create volume from docker host - Error: VM does not belong to any vmgroup
+// 4. Verify docker volume ls does not show any volumes
+// 5. Verify admin cli shows volume with VMGroup set as N/A
+// 6. Create _DEFAULT vmgroup with —default-datastore=_VM_DS
+// 7. Again create new volume - operation should succedd
+// 8. Verify volume from esx and docker host
+func (s *DefaultVMGroupTestSuite) TestDeleteDefaultVmgroupAndVerify(c *C) {
+	misc.LogTestStart(c.TestName())
+	s.volumeName1 = inputparams.GetUniqueVolumeName(defaultVG)
+	s.volumeName2 = inputparams.GetUniqueVolumeName(defaultVG)
```

Comment:
volume name becomes extremely long in that case. e.g. TestDeleteDefaultVmgroupAndVerify_volume_<Some 6-7 digit random number>